ribs/emitters/rankers/_two_stage_objective_ranker.py

```diff
@@ -0,0 +1,50 @@
+"""
+Provides the TwoStageObjectiveRanker
+"""
+import numpy as np
+
+from ribs.emitters.rankers._ranker_base import RankerBase
+
+
+class TwoStageObjectiveRanker(RankerBase):
+    """Ranks the solutions based on the objective values
+
+    This ranker originates in `Fontaine 2020
+    <https://arxiv.org/abs/1912.02400>`_ as OptimizingEmitter.
+    We rank the solutions solely based on their objective values.
+    """
+
+    def rank(self, emitter, archive, solutions, objective_values,
+             behavior_values, metadata, add_statuses, add_values):
+        """Ranks the soutions based on their objective values.
+
+        Args:
+            emitter (ribs.emitters.EmitterBase):
```

Comment:
To avoid repeating docstrings, take a look at how Matplotlib handles duplicates:
- https://github.com/matplotlib/matplotlib/blob/v3.5.2/lib/matplotlib/axes/_axes.py#L4235-L4483
- https://github.com/matplotlib/matplotlib/blob/v3.5.2/lib/matplotlib/pyplot.py#L2814-L2825

Also see Seaborn:
- https://github.com/mwaskom/seaborn/blob/9d8ce6ad4ab213994f0bc84d0c46869df7be0b49/seaborn/relational.py#L643